db/sql/run_tests.sh

```diff
@@ -17,7 +17,7 @@ if [[ $EXIT_CODE -eq 0 ]]; then
     for i in {1..50}; do
         pg_isready -U mathesar -d mathesar_testing && break || sleep 0.5
     done
-    pg_prove --runtests -U mathesar -d mathesar_testing "$@"
+    pg_prove --runtests -U mathesar -d mathesar_testing -v "$@"
```

Comment:
I like this change, are you aware of a way to make only the failing tests verbose?